server/settings/editorial_creators.py

```diff
@@ -39,32 +47,43 @@ class ShotRenameSubmodel(BaseSettingsModel):
 class TokenToParentConvertorItem(BaseSettingsModel):
     _layout = "compact"
     # TODO - was 'type' must be renamed in code to `parent_type`
-    parent_type: str = SettingsField(
-        "Project",
-        enum_resolver=lambda: parent_type_enum
-    )
     name: str = SettingsField(
         "",
-        title="Parent token name",
+        title="Token name",
```

Comment:
Wasn't the `Parent token name` more descriptive? (Just asking, this is out of my scope.)

Reply:
IMO, `parent_token_name` is descriptive when looking closely. 
From user perspective, I'd prefer to see the setting as whole.
Therefore, I can tell that this setting describes a folder path template where each shot will be added.
![image](https://github.com/user-attachments/assets/75e76e29-cb8b-48a1-bc74-aa0366c93b52)

Reply:
Maybe this is better?
```suggestion
        title="Folder token name",
```

Reply:
Like I've wrote, out of my scope, @jakubjezek001 or @robin-ynput might have some ynput.

Reply:
Maybe `Folder token name` is the most meaningful imo. 
But to be honest, this is minimal, I think the description you added and the default examples make it pretty clear.